ee/codegen/src/__test__/nodes/__snapshots__/subworkflow-deployment-node.test.ts.snap

```diff
@@ -13,6 +16,14 @@ class SubworkflowNodeDisplay(BaseSubworkflowDeploymentNodeDisplay[SubworkflowNod
     node_id = UUID("c8f2964c-09b8-44e0-a06d-606319fe5e2a")
     target_handle_id = UUID("f5e6bd33-527a-4ba6-8906-cd5e96a4321c")
     node_input_ids_by_name = {"input": UUID("f62b7511-dd69-4dff-a4fc-718a9db3ceba")}
+    output_display = {
+        SubworkflowNode.Outputs.output_1: NodeOutputDisplay(
+            id=UUID("1"), name="output-1"
+        ),
+        SubworkflowNode.Outputs.output_2: NodeOutputDisplay(
+            id=UUID("2"), name="output-2"
+        ),
+    }
```

Comment:
Shows that we successfully generate static id and name references for output variables.